cpu/atmega256rfr2/include/periph_cpu.h

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2015 HAW Hamburg
+ *               2016 Freie Universität Berlin
+ *		 2017 RWTH Aachen
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_atmega256rfr2
+ * @{
+ *
+ * @file
+ * @brief           CPU specific definitions for internal peripheral handling
+ *
+ * @author          René Herthel <rene-herthel@outlook.de>
+ * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author          Josua Arndt <jarndt@ias.rwth-aachen.de>
+ * @author          Steffen Robertz <steffen.robertz@rwth-aachen.de>
+ */
+
+#ifndef PERIPH_CPU_H
+#define PERIPH_CPU_H
+
+#include "periph_cpu_common.h"
+#include "atmega_regs_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+/**
+ * @name   Length of the CPU_ID in octets
+ * @{
+ */
+#define CPUID_LEN           (8U)
+/** @} */
+
+/**
+ * @name   Available ports on the ATmega256rfr family
+ * @{
+ */
+enum {
+    PORT_B  = 1,        /**< port B */
+    PORT_D  = 3,        /**< port D */
+    PORT_E  = 4,        /**< port E */
+    PORT_F  = 5,        /**< port F */
+    PORT_G  = 6,        /**< port G */
+};
+/** @} */
+
+/**
+ * @name   Atmega256rfr2 timers have up to 3 capture-compare channels (TIMER 1/3) 
+ * -> best suitable for PWM and especially RGB Leds
+ * @{
+ */
+#define TIMER_CHAN          (3U)
+/** @} */
+/* forward declaration of struct to avoid cyclic dependency */
+typedef unsigned int gpio_t;
+/**
+ * @name   GPIO pin not defined
+ * @{
+ */
+#ifndef GPIO_UNDEF
+#define GPIO_UNDEF          (0xFFFF)
+#endif
+/** @}*/
+
+/**
+ * @name   PWM settings struct
+ * @{
+ */
+typedef struct {
+    gpio_t pin;             /**< GPIO pin mapped to this channel */
+    uint8_t cc_chan;        /**< capture compare channel used */
+} pwm_chan_t;
+/** @}*/
+
+
+/**
+ * @name   UART configuration
+ *
+ * The UART devices have fixed pin mappings, so all we need to do, is to specify
+ * which devices we would like to use and their corresponding RX interrupts. See
+ * the reference manual for the fixed pin mapping.
+ *
+ * @{
+ */
+#define UART_NUMOF          (2U)
+
+// UARTo is used for stdio
```

Comment:
cpp style comment, please use C-style comment instead (`/* */`). Maybe this problem is also in other places.